hydra-node/json-schemas/api.yaml

```diff
@@ -1999,6 +2000,8 @@ components:
               $ref: "api.yaml#/components/schemas/HeadId"
             recoverTxId:
               type: string
```

Comment:
note: why this is not $ref: "api.yaml#/components/schemas/TxId" ?

Reply:
I am not sure why is this a `String`. I don't believe I touched this code but this should be changed.

Reply:
It should be.